api/v3/openapi.yaml

```diff
@@ -627,6 +627,45 @@ paths:
         '200':
           $ref: '#/components/responses/GetReplicaResponse'
 
+  /clusters/{cluster_id}/topics/{topic_name}/records:
+    parameters:
+      - $ref: '#/components/parameters/ClusterId'
+      - $ref: '#/components/parameters/TopicName'
+      - $ref: '#/components/parameters/PartitionId'
```

Comment:
I'm not sure why is the `PartitionId` param referred here - isn't it encapsulated in the `ProduceRequest`?

Reply:
That slipped through. :|